<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tbwd">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tasks">
       <attribute name="title">
        <string>传输列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addx">
            <property name="toolTip">
             <string>新建任务(高级)</string>
            </property>
            <property name="text">
             <string>高级新建</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="add">
            <property name="toolTip">
             <string>新建任务(普通)</string>
            </property>
            <property name="text">
             <string>新建</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del">
            <property name="toolTip">
             <string>移除选中任务(含已完成任务的文件)</string>
            </property>
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stop">
            <property name="toolTip">
             <string>暂停选中任务，已成功的不会有反应</string>
            </property>
            <property name="text">
             <string>暂停</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ctne">
            <property name="toolTip">
             <string>续传选中任务，已成功的不会有反应</string>
            </property>
            <property name="text">
             <string>继续</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel">
            <property name="toolTip">
             <string>清除选中任务(不含已完成任务的文件)</string>
            </property>
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restart">
            <property name="toolTip">
             <string>重新开始选中任务，含已成功的</string>
            </property>
            <property name="text">
             <string>重开</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tasktable">
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>10</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>15</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolBox" name="settingtool">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="settings_other">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>197</width>
             <height>144</height>
            </rect>
           </property>
           <attribute name="label">
            <string>杂项设置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>下载成功提示音</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_5">
              <property name="text">
               <string>下载失败提示</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>开机启动</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>桌面快捷方式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>开始菜单快捷方式</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_6">
              <property name="text">
               <string>同一目录只允许运行一份实例</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settings_config">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>89</width>
             <height>89</height>
            </rect>
           </property>
           <attribute name="label">
            <string>下载配置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="configtable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="settings_header">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>982</width>
             <height>584</height>
            </rect>
           </property>
           <attribute name="label">
            <string>请求头配置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPlainTextEdit" name="headeredit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
